DEPENDENCIES.rst

```diff
@@ -0,0 +1,139 @@
+###########
+Cyclus Core
+###########
+
+
+To see user and developer documentation for this code, please visit
+the `Cyclus Homepage`_.
+
+.. contents:: Table of Contents
+   :depth: 2
+
+************
+Dependencies
+************
+
+====================   ==================
+Package                Minimum Version
+====================   ==================
+`Cyclus`               1.4
+====================   ==================
+
+
+***********************
+Installing Dependencies
+***********************
+
+ The only Cycamore dependency is Cyclus. We support 3 main ways to install
```

Comment:
I don't think I saw this info on the cyclus install pages. Should this stuff go there too? Should this just then link to that instead of copying the text?